packages/components/src/form-group/test/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import ControlLabel from '../../control-label';
+import FormGroup, { useFormGroupContextId } from '../index';
+
+// @todo: Refactor this after adding next TextInput component.
+const TextInput = ( { id: idProp, ...props } ) => {
+	const id = useFormGroupContextId( idProp );
+	return <input type="text" id={ id } { ...props } />;
+};
+
+/* eslint-disable no-restricted-syntax */
```

Comment:
What's the reason that this rule got disabled here?

Reply:
It's mad at `id="fname"`

> Do not use string literals for IDs; use withInstanceId instead.

I can add a comment for that